.testing.pylintrc

```diff
@@ -8,7 +8,7 @@
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,ext
```

Comment:
Same comment here as well.

Reply:
This one disables running pylint on anything in the ext/ directory as tom mentioned here.

https://github.com/saltstack/salt/pull/45294#issuecomment-356032611

Is there a different way that I could do that?

Reply:
Oh, I am sorry, I missed that comment and got my reviews mixed up. You're right. ++